script.js

```diff
@@ -1,28 +1,76 @@
-var values  = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A"];
+// During the review I noticed (he also did a disclaimer that he would) that Robin said he was just going to go with it
+// as he codes.  I had assumed that durin the course it would be better "practice" if I used the lessons from the day/week
+// so I tried to make the self and this use compared to what was shown.  Does it matter "how" I code since there is
+// always a different way to do the same thing just depends on what makes sense to me and I use proper commenting?
```

Comment:
Code is just like spoken language in that there are an infinite number of "good" ways to write it. Relevant: 

![bad_code_good_code](https://cloud.githubusercontent.com/assets/6268177/13155779/575fb0ce-d64d-11e5-9111-4e286e1e641e.jpg)